app/views/downloads/download_mac.html.erb

```diff
@@ -31,5 +31,10 @@
   <%= link_to "on kernel.org", "https://www.kernel.org/pub/software/scm/git/" %>.
   The latest version is
   <%= link_to @latest.name, "https://www.kernel.org/pub/software/scm/git/git-#{@latest.name}.tar.gz"%>.
+    
+  <h3>Installing git-gui</h3>
+  <p>If you would like to install <a href="https://git-scm.com/docs/git-gui/">git-gui</a>, git's own 
```

Comment:
> git's own commit gui 

1. should we call it 'GUI client' to be consistent with https://git-scm-pr-1526.herokuapp.com/downloads/guis?

2. should we call it ' graphical user interface to Git'  to be consistent with https://git-scm.com/docs/git-gui/?

3. not sure if we want to say "git's own <something>". We could argue `gitk`would be "git's own GUI"

not sure of what's the best solution here, but wanted to raise the discussion. WDYT?

Reply:
I think that `git's own commit gui` should be replaced with `git's commit GUI`, and add `gitk` as `git's history browser.